### PR TITLE
feat-home-page

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Home Page, Added Basic Routes to all Pages
Created Video Card.
No functionality is Added

## Files to  Check 
- [ ] src/App.js
- [ ] src/index.js
- [ ] src/frontend/components/asideNavigation/AsideNavigation.jsx
- [ ] src/frontend/components/asideNavigation/asideNavigation.css
- [ ] src/frontend/components/navbar/Navbar.jsx
- [ ] src/frontend/components/pageContainer/PageContainer.jsx
- [ ] src/frontend/components/videoCard/VideoCard.jsx
- [ ] src/frontend/pages/home/CategorySlider.jsx

## What not to check

- [ ] package.json 

Preview-Link : https://deploy-preview-1--3am-music.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+					path="/playList"
+					element={<PageContainer page={<Playlist />} />}
+				/>
+				<Route path="/history" element={<PageContainer page={<History />} />} />
+				<Route path="/login" element={<Login />} />
+				<Route path="/signup" element={<Signup />} />
+				<Route
+					path="/userProfile"
+					element={<PageContainer page={<UserProfile />} />}
+				/>
```
Comment (ArjunGTX) [inline src/App.js:37]: You can add a route with `path="*"` for 404 page

File: src/frontend/components/videoCard/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import "./videoCard.css";
+import cardImage from "../../assets/videoImages/demoImage.webp";
+
+function VideoCard() {
+	return (
+		<div className="video__card">
+			<div className="video__card__badge">
+				<i className="far fa-heart fa-2x"></i>
+			</div>
+			<img alt="" src={cardImage} className="video__card-image" />
```
Comment (ArjunGTX) [inline src/frontend/components/videoCard/VideoCard.jsx]: Use a meaningful `alt` for better accessibility

**Review by ArjunGTX — COMMENTED** [review]
Liked how you have structured the files.
Have suggested some changes too

**Review by anshikaa12 — APPROVED** [review]
looks good

File: src/frontend/components/videoCard/VideoCard.jsx (hunk 2 above)
Comment (prathmeshjagtap) [reply src/frontend/components/videoCard/VideoCard.jsx]: done

File: src/App.js (hunk 1 above)
Comment (prathmeshjagtap) [reply src/App.js:37]: yes Added

**Review by ArjunGTX — APPROVED** [review]
Looks good

**Pull request merged**